[2022-04-26 11:27:40,244] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-04-26T11:27:38.097902+00:00 [queued]>
[2022-04-26 11:27:40,289] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this manual__2022-04-26T11:27:38.097902+00:00 [queued]>
[2022-04-26 11:27:40,290] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 11:27:40,299] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-26 11:27:40,300] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 11:27:40,344] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): also_run_this> on 2022-04-26 11:27:38.097902+00:00
[2022-04-26 11:27:40,358] {standard_task_runner.py:52} INFO - Started process 6133 to run task
[2022-04-26 11:27:40,363] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'manual__2022-04-26T11:27:38.097902+00:00', '--job-id', '3', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmprh0tdxgz', '--error-file', '/tmp/tmp0qt9ic5i']
[2022-04-26 11:27:40,364] {standard_task_runner.py:80} INFO - Job 3: Subtask also_run_this
[2022-04-26 11:27:40,526] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this manual__2022-04-26T11:27:38.097902+00:00 [running]> on host c6f211ee236a
[2022-04-26 11:27:40,780] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-04-26T11:27:38.097902+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-26T11:27:38.097902+00:00
[2022-04-26 11:27:40,796] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-26 11:27:40,812] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=manual__2022-04-26T11:27:38.097902+00:00 | dag_run=<DagRun example_bash_operator @ 2022-04-26 11:27:38.097902+00:00: manual__2022-04-26T11:27:38.097902+00:00, externally triggered: True>"']
[2022-04-26 11:27:40,844] {subprocess.py:85} INFO - Output:
[2022-04-26 11:27:40,846] {subprocess.py:89} INFO - run_id=manual__2022-04-26T11:27:38.097902+00:00 | dag_run=<DagRun example_bash_operator @ 2022-04-26 11:27:38.097902+00:00: manual__2022-04-26T11:27:38.097902+00:00, externally triggered: True>
[2022-04-26 11:27:40,847] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-26 11:27:41,418] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20220426T112738, start_date=20220426T112740, end_date=20220426T112741
[2022-04-26 11:27:41,978] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-26 11:27:42,054] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
